lab1/app.css

```diff
@@ -0,0 +1,72 @@
+html {
+	font-family: 'Kavivanar', cursive;
+}
+
+h1 {
+	font-weight: bold;
+}
+
+button {
+    font-size: .8em;
+    border: 1px solid black;
+    border-radius: 12px;
+    background-color: light gray;
+    transition: all ease .3s;
+    float: right;
+}
+
+button:hover {
+    color: white;
+    border: 2px solid white;
+    background-color: black;
+
+}
+
+.story {
+	color: white;
+	background-color: black;
+	margin-bottom: 1em;
+	max-height: 3em;
+	overflow-y: auto;
+}
+
+.story > p {
+	margin: 0;
+	padding: 0;
+
+}
+
+.resource {
+	font-size: 1.3em;
+}
+
+
+.container {
+	display: flex;
+	justify-content: center;
+}
+
+.generators {
+	border: 1px solid black;
+	max-width: 15em;
+	margin: 1em;
+	padding: .5em;
+}
+
+.generator-name {
+	font-size: 1.2em;
+}
+
+.generator-count {
+	float: right;
+	font-size: .5em;
+}
+
+.rate {
+	font-size: .5em;
+}
+
+footer {
+	margin-top: 20em;
+	text-align: center;
+}
```

Comment:
Minor suggestion, although it's tempting to add margin here to the top as high number so it can push the item down. It's better off to not do that because this may look weird as the content grows more in the body area.